corehq/apps/linked_domain/remote_accessors.py

```diff
@@ -126,7 +126,8 @@ def fetch_remote_media(local_domain, missing_media, remote_app_details):
             item['multimedia_id'], item['upstream_media_id'] = media_item._id, item['multimedia_id']
         else:
             media_item._id = item['multimedia_id']
-        media_item.attach_data(content, original_filename=filename)
+        if not any(a in media_item.aux_media for a in media_item.aux_media if a.uploaded_filename == filename):
```

Comment:
I *think* this is the check I need to make here, since it seems like `AuxMedia` exists just to ensure the app's reference to the filename is valid.

Also thought about extracting this line along with 124 into named functions for clarity (probably `_media_exists_locally` and `_needs_data_attached`, but I wasn't sure if it would be helpful or needed for the one-liners that they are.